src/core/cancellation_middleware.py

```
@@ -24,16 +24,20 @@ async def on_call_tool(self, context: MiddlewareContext, call_next):
         start_time = time.perf_counter()
         
         try:
-            logger.debug(f"Starting tool call: {getattr(context, 'method', 'unknown')}")
+            tool_method = getattr(context, 'method', 'unknown')
+            logger.info(f"🛡️  CANCELLATION MIDDLEWARE: Starting tool call: {tool_method}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)